client/src/components/Common/ErrorPluginTracker.vue

```diff
@@ -0,0 +1,16 @@
+<script setup lang="ts">
+import { getGalaxyInstance } from "@/app";
+import { computed } from "vue";
+
+const errorReportingAvailable = computed(() =>{
+    const Galaxy = getGalaxyInstance();
+    if (!!Galaxy.Sentry.isInitialized) {
```

Comment:
This just means we have a sentry client that is able to submit client errors, but that doesn't mean entirely expected things like this error are captured in sentry.